src/kernel/native-object.cc

```diff
@@ -748,6 +748,15 @@ NATIVE_FUNCTION(NativesObject, StopVideoLog) {
     GLOBAL_boot_services()->logger()->DisableVideo();
 }
 
+NATIVE_FUNCTION(NativesObject, SetTime) {
+  PROLOGUE;
+  USEARG(0);
+  VALIDATEARG(0, NUMBER, "argument 0 is not a number value");
+  uint64_t val = (arg0.As<v8::Number>())->Value();
```

Comment:
:+1: nice! `As<v8::Numner>` is going to be faster because `->NumberValue()` calls into js internally